src/app/src/components/FilterSidebarSearchTab.jsx

```diff
@@ -83,12 +89,50 @@ const CONTRIBUTOR_TYPES = 'CONTRIBUTOR_TYPES';
 const LISTS = 'LISTS';
 const COUNTRIES = 'COUNTRIES';
 const PARENT_COMPANY = 'PARENT_COMPANY';
+const FACILITY_TYPE = 'FACILITY_TYPE';
+const PROCESSING_TYPE = 'PROCESSING_TYPE';
+const mapFacilityTypeOptions = (fPTypes, pTypes) => {
```

Comment:
`mapFacilityTypeOptions` and `mapProcessingTypeOptions` ensure that facility types and processing types are syncing up so no contradicting results